corehq/apps/auditcare/utils/export.py

```diff
@@ -114,3 +77,78 @@ def write_export_from_all_log_events(file_obj, start, end):
     writer.writerow(['Date', 'Type', 'User', 'Domain', 'IP Address', 'Action', 'Resource', 'Description'])
     for event in get_all_log_events(start, end):
         write_generic_log_event(writer, event)
+
+
+def get_date_range_where(start_date, end_date):
+    where = {}
+    if start_date:
+        where["event_date__gt"] = start_date.date()
+    if end_date:
+        where["event_date__lt"] = end_date.date() + timedelta(days=1)
```

Comment:
This was copied from pre-existing code, and I didn't think about that too hard since I think this works.

I think `event_date__lte` will not work since this is coercing `end_date` to a `date` (something like `2020-03-23`) and `event_date` is a timestamp (includes a time). My understanding is that when SQL evaluates `end_date <= '2020-03-23'::datetime` (or whatever it would do in that case), it coerces `'2020-03-23'` to `'2020-03-23 00:00:00.000'`, so it would get the wrong answer for `end_date` values like `2020-03-23 12:34:56.789`.

Could revisit this in a follow-up PR, although there are other things that seem more valuable to work on since I think this code works as is.